general_itests/steps/deployments_json_steps.py

```diff
@@ -172,3 +173,10 @@ def step_impl_then_correctly_tagged(context):
     expected_formatted_tag = format_tag(expected_tag)
     assert expected_formatted_tag in context.test_git_repo.refs
     assert context.test_git_repo.refs[expected_formatted_tag] == context.expected_commit
+
+
+@then(u'the repository should not have old style branches')
+def step_impl_then_no_old_style_branches(context):
```

Comment:
I don't know how I feel about this 'making sure we no longer do things the old way' testing. In the future this test won't have any context outside of old versions of paasta.